src/common/types.hpp

```diff
@@ -41,8 +41,6 @@
 #include <string>
 #include <vector>
 
-#include <openthread/ip6.h>
```

Comment:
Any background on the decision to not allow including openthread headers in otbr headers? @Moandor-y @bukepo

Reply:
The goal is to keep the interface stable for internal projects. As you know, OpenThread itself iterates rapidly.